R/makeAggregatedDataset.R

```diff
@@ -62,7 +65,7 @@
 #' @family loading
 
 makeAggregatedDataset <- function(source.dir, ncml.file, file.ext = "nc", aggr.dim = "time",
-                        pattern = NULL, recursive = FALSE, verbose = TRUE) {
+                        pattern = NULL, recursive = FALSE, verbose = TRUE, timeUnitsChange = FALSE) {
```

Comment:
timeUnitsChange = TRUE

Reply:
Hi @cofinoa are you really sure making default TRUE will not have a bad effect on files which do NOT have a relative time axis? I asked the netcdf-java mailing list an opinion on this.

Reply:
Here is the answer from my question in the netcdf-java mailing list:

There is no harm in having that.  It's been used with all kinds of datasets in Unidata IDV for years. 

So @cofinoa do you want me to make the changes & test?